enterprise/payara.common/nbproject/org-netbeans-modules-payara-common.sig

```diff
@@ -1340,7 +1340,7 @@ meth public org.netbeans.modules.payara.common.PayaraInstanceProvider getInstanc
 meth public org.netbeans.modules.payara.common.PayaraJvmMode getJvmMode()
 meth public org.netbeans.modules.payara.common.parser.DomainXMLChangeListener getDomainXMLChangeListener()
 meth public org.netbeans.modules.payara.tooling.data.PayaraAdminInterface getAdminInterface()
-meth public org.netbeans.modules.payara.tooling.data.PayaraVersion getVersion()
+meth public org.netbeans.modules.payara.tooling.data.PayaraPlatformVersionAPI getVersion()
```

Comment:
Alas, to be binary backward compatible, you have to keep the old method signature and introduce new method:
```
PayaraPlatformVersionAPI getVersionApi()
```
You can choose the method name, of course. Technically it would even be `getVersion()`, but you are going to have problems to compile Java source that contains two methods without arguments with different return types.